### ISSUE TITLE
CRA | 'stryker run' doesn't respect .env files

### ISSUE THREAD

**Issue by Lighttree** [stryker-mutator/stryker-js#3022]
**Summary**

`Create React App` supports adding environment variables listed in .env files. [Read more](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env)
A `.env.test` file may contain multiple vars that the `react-scripts test` command respects during a test run.
It seems that the `stryker run` command somehow misses this and executes tests without `vars`.

This was discovered as my tests are using `@testing-library/react` for UI testing, and `REST API` mocked using `MirageJS`, so I'm using some `env` vars for tests, to enable this.

**Stryker config**

```json
{
    "$schema": "./node_modules/@stryker-mutator/core/schema/stryker-schema.json",
    "_comment": "This config was generated using 'stryker init'. Please see the guide for more information: https://stryker-mutator.io/docs/stryker-js/guides/react",
    "testRunner": "jest",
    "reporters": ["progress", "clear-text", "html"],
    "coverageAnalysis": "off",
    "jest": {
        "projectType": "create-react-app"
    },
    "tempDirName": "stryker-tmp"
}
```

**Test runner config**

Standard `CRA` config.

**Stryker environment**

```
@stryker-mutator/core@5.2.2
@stryker-mutator/jest-runner@5.2.2
```

```
react-scripts@4.0.3
```

**Test runner environment**

```shell
"test": "react-scripts test"
```

Executing by `yarn test`

Standard `CRA` config.

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | v14.16.1
| npm              | 6.14.12
| Operating System | MacOS

**Add stryker.log**

**Comment by nicojs** [comment]
Thanks for opening this issue. I've found the cause and prepared a fix.

**Comment by nicojs** [comment]
This feature just got released for Stryker 5.3 please give it a try
